qt-qml/src/debug/file-finder.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2025 The Qt Company Ltd.
+// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import { isEmpty } from 'lodash';
```

Comment:
is it worth to include all of lodash to save a `matchedFilePaths.length() == 0` ?